management-api/src/main/java/com/afh/skytouch/managment/controllers/GenericProductController.java

```diff
@@ -14,20 +16,33 @@
 @RequestMapping("/genericProduct")
 public class GenericProductController {
 
-    @GetMapping("/{uuid}")
-    public ResponseEntity<?> getProduct(@PathVariable UUID uuid){
-        return ResponseEntity.ok().build();
+    private static final String productAtributeName = "product";
+    private static final String creationPage = "create-product";
+    private static final String homePage = "home";
+    private static final String showProductsPage = "list-products";
+
+    @Autowired
+    ProductSender sender;
```

Comment:
Implement constructor or setter based injection, also, the field could be private I guess